procfile.dev

```diff
@@ -0,0 +1,2 @@
+web: bin/rails server -p 3070
```

Comment:
I think this should be port 3000, as per the comment in [the docs](https://docs.publishing.service.gov.uk/manual/migrate-to-dart-sass-from-libsass.html#watch-mode):

> Note, the port number should match the one previously specified, in startup.sh and or docker-compose.yml, for running your Rails server.

Reply:
Addressed in https://github.com/alphagov/support/pull/1253/commits/610e25a519a80a6db8e3918bd38c5ff2e1b7d3cc